defaults/main.yml

```diff
@@ -33,13 +39,13 @@ alpaca_settings:
     settings:
       input.stream: activemq:topic:fedora
       triplestore.reindex.stream: activemq:queue:triplestore.reindex
-      triplestore.baseUrl: http://localhost:8080/bigdata/namespace/islandora/sparql
+      triplestore.baseUrl: "{{ alpaca_triplestore_base_url }}/namespace/islandora/sparql"
```

Comment:
If we are going part way, why not remove the assumption that all triplestores will use the "islandora" namespace.